lib/pay/braintree/billable.rb

```diff
@@ -31,9 +31,9 @@ def braintree_customer
           result.customer
         end
       rescue ::Braintree::AuthorizationError
-        raise BraintreeAuthorizationError
+        raise Pay::Braintree::AuthorizationError
```

Comment:
This would be a breaking change. We would have to raise the old error with the deprecation warning until the next major release where we could replace this.

I very much like the new Error namespacing, so I definitely want to do that. If we `raise` the old error classes, we can replace this in the next major version bump.

Reply:
Yeah makes sense, I may have come to a similar thinking over the christmas break. So this was a good little reminder. 👍